server/activityItem/model.ts

```diff
@@ -0,0 +1,11 @@
+export default (sequelize, dataTypes) => {
+	return sequelize.define('ActivityItem', {
+		id: sequelize.idType,
+		kind: { type: dataTypes.TEXT, allowNull: false },
+		pubId: { type: dataTypes.UUID },
+		payload: { type: dataTypes.JSONB },
+		communityId: { type: dataTypes.UUID, allowNull: false },
+		actorId: { type: dataTypes.UUID, allowNull: false },
+		collectionId: { type: dataTypes.UUID },
```

Comment:
Oh, yeah, could we add a field here:

```js
timestamp: {
    type: dataTypes.DATETIME,
    defaultValue: sequelize.Sequelize.NOW,
    allowNull: false,
}
```
been thinking that we probably want to have an explicit statement of the moment in time these items refer to besides the `createdAt` and `updatedAt` that Sequelize creates, since some will be created after the fact with a backfill, and might but updated later in the future.